lua/cfc_voice_channels/core/sv/sv_channel_control.lua

```diff
@@ -55,14 +55,15 @@ function cfc_voice:CreateChannel(caller, name, password)
         return 
     end
 
-    cfc_voice.Channels[table.Count(cfc_voice.Channels) + 1] = {
+    local i = table.Count(cfc_voice.Channels) + 1
+    cfc_voice.Channels[i] = {
```

Comment:
This functionality could be extracted into its own method.

`storeChannel`, `saveChannel`, `trackChannel`, something like that

Reply:
Ooh, ok i'll probably do that in a different PR maybe?

Reply:
you can - it'd be really easy to do quickly